Description: avoid crate seahash
 Crate seahash is used only in a benchmark
 that cannot build with stable rust.
 .
 While at it, disable all benchmarks,
 as none of them can buil with stable rust.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-04-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: http/Cargo.toml
===================================================================
--- http.orig/Cargo.toml
+++ http/Cargo.toml
@@ -22,21 +22,21 @@ categories = ["web-programming"]
 license = "MIT/Apache-2.0"
 repository = "https://github.com/hyperium/http"
 
-[[bench]]
-name = "header_map"
-path = "benches/header_map/mod.rs"
-
-[[bench]]
-name = "header_name"
-path = "benches/header_name.rs"
-
-[[bench]]
-name = "header_value"
-path = "benches/header_value.rs"
-
-[[bench]]
-name = "uri"
-path = "benches/uri.rs"
+#[[bench]]
+#name = "header_map"
+#path = "benches/header_map/mod.rs"
+
+#[[bench]]
+#name = "header_name"
+#path = "benches/header_name.rs"
+
+#[[bench]]
+#name = "header_value"
+#path = "benches/header_value.rs"
+
+#[[bench]]
+#name = "uri"
+#path = "benches/uri.rs"
 [dependencies.bytes]
 version = "1"
 
@@ -51,9 +51,6 @@ version = "0.3"
 [dev-dependencies.indexmap]
 version = "1.0"
 
-[dev-dependencies.seahash]
-version = "3.0.5"
-
 [dev-dependencies.serde]
 version = "1.0"
 
